### ISSUE TITLE
Some game cheats not downloading

### ISSUE THREAD

**Issue by game174master** [Fledge68/WiiFlow_Lite#387]
For some reason, whenever I try to download cheats for games, it doesn't work. WiiFlow keeps producing a "download failed" message. and whenever I try to manually put in the cheat codes(Downloaded from gamehacking.org), it shows the cheats, but doesn't boot the game. It's just a black screen. But for some reason, some games can download cheat codes, and others can't, for example, I can download the Mario Galaxy 2 cheat codes from WiiFlow, but not Mario Galaxy.

v5.5.4-1
vWii

**Comment by eku** [comment]
Did you try the [help forum](https://gbatemp.net/threads/wiiflow-lite.422685/) at gbatemp?

**Comment by gingerbeardman** [comment]
If the game rip is modified in some ways then cheat codes will fail to be downloaded.

Make sure your game rip is clean and verified.

**Comment by game174master** [comment]
The thing is that I clean ripped Mario Galaxy and other games.

**Comment by gingerbeardman** [comment]
Then head over to the help forum.

**Comment by game174master** [comment]
how do we ask a question in GBAtemp?

**Comment by gingerbeardman** [comment]
Simply post on the thread linked above.

**Comment by game174master** [comment]
How?

**Comment by MayanKoyote** [comment]
> How? The only button that I can see is the reply button to a post from 2016.

Don’t worry, no one will answer you anything useful there anyway. There has long been nothing more than a garbage dump.

**Comment by gingerbeardman** [comment]
@MayanKoyote this is quite disrespectful, there are many answers in there and, let's not forget, it's an open source project and volunteer community so you should set your expectations and entitlement accordingly.

**Comment by gingerbeardman** [comment]
@game174master right, the support thread has been active since 2016, is currently at 228 pages of replies, and the most recent reply was 13 minutes ago as of me writing this post. It's the recommended support method for this project.

**Comment by MayanKoyote** [comment]
I haven't laughed so much for a long time.
Remember once and for all one simple thing: do not confuse red, warm, wet and soft. These are always different things.

As for this particular situation, please read the analysis of this specific situation:

1. A person spends time successfully searching the Internet for the corresponding repository on Github. Describes the problem (yes, as best he can, but he does it clearly, which is already rare in the modern world). This alone is worthy of respect in our modern days.
Yes, his question is simple and the answer could be found with the help of further search, but this technology (search) has already been lost in the modern world.
2. The answer to this fairly simple question could take up at most one single line of text on the screen, but!:
3. Instead of it, you are sending people to some kind of 3rd party forum that no one needs (an forum! Nowadays, in modern times!). Those to the place where one need to specifically register again and then figure out what to do with these 256 pages and over 9000 of messages of various nonsense in that topic.

Ten years ago this would have been acceptable. But not nowadays.

This is, at a minimum, a lack of respect for the user. And considering what kind of forum this is, this can even be regarded as an insult.

And instead of making indignant bearded faces like in memes, look around at the world around you, at the fact that it has changed in the last 10 years, at the fact that people don’t need your rotten 3rd party forums with 256 pages of unnecessary nonsense consisting of over 9000 messages.

Stop making people laugh and disgrace yourself.
Start Discord at least.
Everyone in the modern world has it in phones already and does not raise any questions how it works.
It's simple. After all, all correct solutions to problems are always simple.

**Comment by gingerbeardman** [comment]
I think you forgot to give the solution in your condescending wall of text 🙄

**Comment by game174master** [comment]
I had completely forgotten about this until I saw the closed notification. Tried it today and it works perfectly.
